Userland/Games/Solitaire/Game.cpp

```diff
@@ -144,7 +152,12 @@ void Game::keydown_event(GUI::KeyEvent& event)
 
     if (event.shift() && (event.key() == KeyCode::Key_F12))
         start_game_over_animation();
-    else if (event.shift() && (event.key() == KeyCode::Key_F11))
+    else if (event.key() == KeyCode::Key_Tab)
+        auto_move_eligible_cards_to_stacks();
+    else if (event.key() == KeyCode::Key_Space) {
+        draw_cards();
+        invalidate_layout(); // FIXME: Stock stack won't render properly after draw_cards() without this
+    } else if (event.shift() && (event.key() == KeyCode::Key_F11))
```

Comment:
I think when adding parentheses to one of the `else if` blocks, we should add them to the other blocks in the chain as well.

Reply:
Did you mean curly braces and not parentheses?

Reply:
Oops :facepalm: I did mean braces, sorry

Reply:
No worries! I've updated the code to use curly braces, and will stick to that coding style going forward. Thanks!

Reply:
This doesn't seem to be resolved - this should be:

```c++
} else if (event.shift() && (event.key() == KeyCode::Key_F11)) {
    dump_layout();
}
```

i.e. if any of the if/else blocks needs curly braces, all of them should get them. The parentheses around the key equality comparison are not needed though :)

Reply:
Thanks, good catch on that last curly brace, overlooked it when I was updating the others. 🤦

The parentheses for key comparison were existing to the codebase but I have cleaned that up while I'm editing these lines anyways.